source/features/hide-useless-comments.tsx

```diff
@@ -15,8 +15,28 @@ function unhide(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
 	event.delegateTarget.parentElement!.remove();
 }
 
+function hideComments(commentsToHide: any[]): void {
+	for (const comment of commentsToHide) {
+		comment.hidden = true;
+		comment.classList.add('rgh-hidden-comment');
+	}
+}
+
 function init(): void {
-	let uselessCount = 0;
+	const commentsToHide: any[] = [];
+
+	// Get all "similar comment" boxes
+	const similarCommentBoxes = select.all('.Details-element');
+
+	for (const similarCommentsBox of similarCommentBoxes) {
+		const similarCommentBtn: HTMLElement = similarCommentsBox.querySelectorAll('.Details-content--closed')[0] as HTMLElement;
```

Comment:
`select()` can take a root element for the query as the second argument:
```suggestion
		const similarCommentBtn = select('.Details-content--closed', similarCommentsBox);
```
Same for the next line.

Reply:
There are several querySelectors while there should only be one at the top (and only one loop). Once my other comment is applied, this should become clearer